core/include/swatch/core/AbstractMetric.hpp

```diff
@@ -35,6 +36,19 @@ enum Status {
   kNonCritical,
   kDisabled,
 };
+
+const boost::unordered_map<std::string, Status> StringToStatus= {
```

Comment:
maybe better to use a std::map. boost::unordered_maps are efficient when the number of lookups is large.
Also, will this initialisation syntax work when compiled w/o c++0x flag?